packages/package-a/tsconfig.json

```diff
@@ -1,22 +1,11 @@
 {
-    "compilerOptions": {
-      "module": "commonjs",
-      "esModuleInterop": true,
-      "declaration": true,
-      "outDir": "dist",
-      "noImplicitAny": true,
-      "preserveConstEnums": true,
-      "removeComments": true,
-      "sourceMap": true,
-      "target": "es5",
-      "lib": [
-        "dom",
-        "es2015"
-      ]
-    },
-    "exclude": [
-      "node_modules",
-      "dist",
-      "tests"
-    ]
-  }
\ No newline at end of file
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "outDir": "dist"
+  },
+  "exclude": [
+    "node_modules",
+    "dist",
+    "tests"
+  ]
+}
```

Comment:
can we have newlines at the end of files plz?

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Oops... My bad